src/util/Conversions.h

```diff
@@ -159,16 +170,25 @@ string convertIndexWordToValueLiteral(const string& indexWord) {
   if (startsWith(indexWord, VALUE_FLOAT_PREFIX)) {
     if (endsWith(indexWord, "F")) {
```

Comment:
If we use `indexWord.back()` (which is save as long as `VALUE_FLOAT_PREFIX` is non empty) we could use the new NumericType enumeration for the comparison here.